modules/learn/pages/data/durability.adoc

```diff
@@ -58,258 +62,303 @@ The correspondences are as follows:
-The constraints are related to the _level_ of durability specified, the nature of the outage, and the number of replicas.
-The guarantees and associated constraints are stated on this page, below.
+After Couchbase Server commits the durable write, it notifies clients that the write succeeded.
+From the point of commitment until the new data has fully propagated throughout the cluster, Couchbase Server guarantees protection from data loss within some constraints.
+These constraints depend on the durability level you set, the type of outage, and the number of replicas.
+You can find the guarantees and related constraints in the following sections.
 
 [#replica-count-restriction]
 === Replica-Count Restriction
 
-Couchbase-Server durability supports buckets with up to _two_ replicas.
-It does _not_ support buckets with _three_ replicas.
-If a durable write is attempted on a bucket that has been configured with three replicas, the write fails with an `EDurabilityImpossible` message.
+Couchbase Server supports durability for buckets with up to two replicas.
+You cannot use durability with buckets that have three replicas.
+If you try to perform a durable write on a bucket with three replicas, Couchbase Server returns an `EDurabilityImpossible` error.
 
 [#protection-guarantees-and-automatic-failover]
 === Protection Guarantees and Automatic Failover
 
-xref:learn:clusters-and-availability/automatic-failover.adoc[Automatic Failover] removes a non-responsive node from the cluster automatically, following an administrator-configured timeout.
-Active vBuckets thereby lost are replaced by the promotion of replica vBuckets, on the surviving nodes.
-The maximum number of sequential automatic failovers is configurable by the administrator.
+xref:learn:clusters-and-availability/automatic-failover.adoc[Automatic Failover] removes a non-responsive node from the cluster after a configurable timeout.
+When Couchbase Server fails over a node, it promotes replica vBuckets on surviving nodes to replace the lost active vBuckets.
+You can configure the maximum number of sequential automatic failovers.
 
-In cases where commitment based on _persistToMajority_ has occurred, but no further propagation of the new data across the cluster has yet occurred, automatic failover of the nodes containing the new data results in the data's loss &#8212; since no updated replica vBucket yet exists elsewhere on the cluster.
+Automatic failovers can cause the loss of a durable write.
+If you perform a successful durable write using `persistToMajority` but the data has not yet propagated in the cluster beyond the majority, automatic failover of the majority nodes causes data loss.
+This loss happens because there is no updated replica vBucket in the cluster.
```

Comment:
This loss happens because there are not enough up to date replica vBuckets in the cluster.